src/auro-checkbox.scss

```diff
@@ -166,16 +206,14 @@ fieldset {
     pointer-events: none;
     z-index: 1;
 
-    top: 6px;
-    left: 18px;
-    width: 6px;
-    height: 18px;
-    border-style: solid;
-    border-color: var(--auro-color-base-white);
-    border-width: 2px;
-    border-top-width: 0;
-    border-left-width: 0;
-    transform: rotate(45deg);
+    top: 4px;
+    left: var(--cbxLocation);
+    width: 20px;
+    height: 20px;
+    background-repeat: no-repeat;
+    background-color: var(--auro-color-base-white);
+    -webkit-mask-image: url('https://unpkg.com/@alaskaairux/icons@latest/dist/icons/interface/check-lg.svg');
+    mask-image: url('https://unpkg.com/@alaskaairux/icons@latest/dist/icons/interface/check-lg.svg');
```

Comment:
I have concerns about using the CDN resource for this. It would be more sturdy if we shipped the check SVG with the ./src of the repo. 

We could have a build step that copies the SVG out of the auro Icons package as an insurance policy so this SVG stays up to date with Auro Icons as it changes. 

It does require a release to update the icon, but that's a small price to pay versus the SVG failing because the CDN is having a bad day.